docs/modules/operation/pages/service-assurance/polling-packages.adoc

```diff
@@ -65,11 +64,11 @@ Each polling package contains an RRD definition.
     <rra>RRA:MIN:0.5:288:366</rra><6>
 </rrd>
 ----
-<1> Polling interval for all services in this polling package is reflected in the step of size 300 seconds.
-    All services in this package have to be polled in a 5-minute interval, otherwise response time measurements are not persisted correctly.
+<1> Polling interval for all services in this polling package is reflected in the step size of 300 seconds.
+    All services in this package must be polled in a five-minute interval, otherwise response time measurements are not persisted correctly.
```

Comment:
```suggestion
    All services in this package must be polled on a five-minute interval, otherwise response time measurements are not persisted correctly.
```